tnefparse/cmdline.py

```diff
@@ -47,18 +47,12 @@
          help="extract a json dump of the tnef contents")
 
 
-def tnefparse() -> None:
+def tnefparse(argv=None) -> None:
     "command-line script"
 
-    if len(sys.argv) == 1:
-        parser.print_help()
-        sys.exit(1)
```

Comment:
This guard is still necessary.

Although `argparse` shows some kind of help when called without arguments with your change..

```
❯ tnefparse
usage: tnefparse [-h] [-o] [-a] [-p PATH] [-b] [-hb] [-rb] [-l {DEBUG,INFO,WARN,ERROR}] [-c] [-d] file [file ...]
tnefparse: error: the following arguments are required: file
```
... `print_help` show this much more user friendly version:
```
❯ tnefparse
usage: tnefparse [-h] [-o] [-a] [-p PATH] [-b] [-hb] [-rb] [-l {DEBUG,INFO,WARN,ERROR}] [-c] [-d] file [file ...]

Extract TNEF file contents. Show this help message if no arguments are given.

positional arguments:
  file                  space-separated list of paths to the TNEF files

optional arguments:
  -h, --help            show this help message and exit
  -o, --overview        show (possibly long) overview of TNEF file contents
  -a, --attachments     extract attachments, by default to current dir
  -p PATH, --path PATH  optional explicit path to extract attachments to
  -b, --body            extract the body to stdout
  -hb, --htmlbody       extract the HTML body to stdout
  -rb, --rtfbody        extract the RTF body to stdout
  -l {DEBUG,INFO,WARN,ERROR}, --logging {DEBUG,INFO,WARN,ERROR}
                        enable logging by setting a log level
  -c, --checksum        calculate checksums (off by default)
  -d, --dump            extract a json dump of the tnef contents
```

The reason the tests are green is once again missing coverage.

We are already at a very nice 93% coverage, but I think it would be a good thing to get it up to 100%.

Reply:
Right, then it belongs back there.